_episodes/13-scheduler.md

```diff
@@ -314,8 +314,8 @@ minutes.
 {{ site.sched.comment }} {{ site.sched.flag.time }} 00:00:30
 
 echo -n "This script is running on ... "
-sleep 120 # time in seconds
 hostname
+sleep 120 # time in seconds
```

Comment:
Either this reordering or remove the `-n` option to `echo`, leaving it resulting in mega long line